### PR TITLE
Feature/add emr serverless alerts

### PR DESCRIPTION
**Pull request by aryna-yukhnevich**
image

tests
image
image

### CODE REVIEW

File: tests/src/lib/event_mapper/test_emr_serverless_event_mapper.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        "applicationId": EMR_APP_ID,
+        "jobRunId": "00flfo8g80vlko17",
+        "name": JOB_NAME,
+        "createdAt": str(datetime(2000, 1, 1, 0, 0, 0)),
+        "updatedAt": str(datetime(2000, 1, 1, 1, 0, 0)),
+        "state": "FAILED",
+        "stateDetails": "Job failed",
+        "jobDriver": {
+            "sparkSubmit": {
+                "entryPoint": "s3://s3-salmonts-emr-scripts-dev/job2_failure.py"
```
Comment (natmasslove) [inline tests/src/lib/event_mapper/test_emr_serverless_event_mapper.py:23]: I guess, s3 bucket name doesn't matter here, so maybe let's use something "not real"

Comment (natmasslove) [reply tests/src/lib/event_mapper/test_emr_serverless_event_mapper.py:23]: I'm wondering if it makes sense to add EMR serverless items into sample config

Comment (aryna-yukhnevich) [reply tests/src/lib/event_mapper/test_emr_serverless_event_mapper.py:23]: sure, let me update this entryPoint and sample config
thank you for your comments!

**Review by natmasslove — APPROVED** [review]
Looks very good, thanks!

**Pull request merged**